docs/get-started.md

```diff
@@ -113,6 +115,12 @@ Follow the steps and login to your CapRover instance. When prompted to enter the
 > **NOTE**: **It will not be possible to carry through with the `caprover serversetup` if you've already forced https on your CapRover instance.**
 > In such case go straight to logging in with the `caprover login` command. To change the password go to the settings menu in the app.
 
+### With the web interface (doesn't require npm)
+
+Visit `http://[IP_OF_YOUR_SERVER]:3000` and start by changing the default password (`captain42`).
```

Comment:
You should never change the password before enabling and enforcing HTTPS, as HTTP is an insecure channel.
This is the correct order of the steps:

1- Login
2- Enable HTTPS
3- Force HTTPS
4- Change password

Reply:
Oops, thanks! I added setting the root domain because if I remember correctly it's required to enable HTTPS.